std/range/package.d

```diff
@@ -6911,21 +6911,27 @@ Indexed!(Source, Indices) indexed(Source, Indices)(Source source, Indices indice
 
 /**
 This range iterates over fixed-sized chunks of size $(D chunkSize) of a
-$(D source) range. $(D Source) must be a forward range. $(D chunkSize) must be
+$(D source) range. $(D Source) must be an input range. $(D chunkSize) must be
 greater than zero.
 
 If $(D !isInfinite!Source) and $(D source.walkLength) is not evenly
 divisible by $(D chunkSize), the back element of this range will contain
 fewer than $(D chunkSize) elements.
 
+If `Source` is an input range but not a forward range, the resulting range and
+chunks will be single-pass only: iterating over `front` will shrink the chunk
```

Comment:
"If Source is a forward range, the resulting chunks will be forward ranges as well. Otherwise (i.e. Source is an input range), the resulting chunks will be input ranges consuming the same input."